### PR TITLE
DOC: update NEP status for accepted/finished NEPs

### PR DESCRIPTION
**Pull request by rgommers**
This is just minor overdue admin on NEP status.

`[ci skip]`

### CODE REVIEW

File: doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst [hunk 1]
```diff
@@ -5,11 +5,11 @@ NEP 35 — Array Creation Dispatching With __array_function__
 ===========================================================
 
 :Author: Peter Andreas Entschev <pentschev@nvidia.com>
-:Status: Draft
+:Status: Accepted
 :Type: Standards Track
 :Created: 2019-10-15
 :Updated: 2020-11-06
-:Resolution:
+:Resolution: https://mail.python.org/pipermail/numpy-discussion/2020-November/081193.html
```
Comment (seberg) [inline doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst:12]: Oh, I guess we should add a `.. note::` that this is considered experimental API as of NumPy 1.20?

Comment (rgommers) [reply doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst:12]: It has that note in function docstrings:

image

Is that true though? Did we decide it was experimental, and till when?

Comment (seberg) [reply doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst:12]: Well, we didn't say it wasn't experimental either, I guess.  I think e.g. the `like=` argument being strict might still have some re-thinking with real world experience. @pentschev what is your thought?

For simplicity, just take this one out and merge the rest for now?

Comment (pentschev) [reply doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst:12]: I don't have any particular preferences right now. We have some support for it in Dask that got merged last week (https://github.com/dask/dask/pull/6738) with some good feedback already (granted, mostly internal). From my personal perspective it seems to indeed resolve the cases I've been able to work on so far and the work on strict/non-strict is mainly a syntactic one that doesn't affect usability that couldn't be handled properly within Dask, for example. Accepting the NEP feels like a step in the right direction, but if you prefer we can wait until closer to 1.21 to discuss the status once again.

Comment (rgommers) [reply doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst]: Thanks, I've changed it back to Draft for now, and added the link to the mailing list to _Discussion_.

**Comment by seberg** [comment]
Thanks Ralf!

**Pull request merged**